Core/Page Object/loginPage.ts

```diff
@@ -0,0 +1,23 @@
+import { Locator, Page } from '@playwright/test';
+import Users from '../../Core/Constant';
+import {BasePage} from "./Base";
```

Comment:
BasePage is imported but not used. Either extend LoginPage from this class or remove the import